core/src/main/java/com/scalar/db/storage/rpc/GrpcUtils.java

```diff
@@ -0,0 +1,17 @@
+package com.scalar.db.storage.rpc;
+
+import io.grpc.ManagedChannel;
+import io.grpc.netty.NettyChannelBuilder;
+
+public final class GrpcUtils {
+
+  private GrpcUtils() {}
+
+  public static ManagedChannel createChannel(GrpcConfig config) {
+    NettyChannelBuilder builder =
+        NettyChannelBuilder.forAddress(config.getHost(), config.getPort()).usePlaintext();
+    config.getMaxInboundMessageSize().ifPresent(builder::maxInboundMessageSize);
+    config.getMaxInboundMetadataSize().ifPresent(builder::maxInboundMetadataSize);
+    return builder.build();
+  }
```

Comment:
Extracted a method to create a channel. Here we set the configurations to a channel.